packages/doc-site/src/components/Dial.md

```diff
@@ -0,0 +1,150 @@
+# Overview
+
```

Comment:
expecting a paragraph that explains what this component is (check out https://synchrocharts.com/#/Components/KPI for an example)

Reply:
Done